apps/web/components/eventtype/EventAvailabilityTab.tsx

```diff
@@ -17,13 +18,6 @@ import { ExternalLink, Globe } from "@calcom/ui/components/icon";
 
 import { SelectSkeletonLoader } from "@components/availability/SkeletonLoader";
 
-export type AvailabilityOption = {
-  label: string;
-  value: number;
-  isDefault: boolean;
-  isManaged?: boolean;
-};
```

Comment:
Also moved to the features/lib to be able to use it from there too and avoid crossing boundaries between web app and the library